apis/python/build-wheels.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
```

Comment:
Can we use `cibuildwheel` instead of this script?

Reply:
Good suggestion. These are VCF's first steps towards building wheels and suggestions for standard processes are welcome.